packages/@react-spectrum/utils/src/Slots.tsx

```diff
@@ -17,3 +17,18 @@ export let SlotContext = React.createContext(null);
 export function useSlotProvider() {
   return useContext(SlotContext) || {};
 }
+
+export function ClearSlots(props) {
```

Comment:
what is this for?

Reply:
Was thinking, slots is a bit pervasive, in a dialog, it applies specific styles to the Divider. If someone wanted to make a two column layout in their content and use a Divider, they'd need to do some work to get rid of the class name the dialog applies.
My thought was to start by just having the semantic element <Content> always clear the slots context because I don't think we typically want to reach into that region of anything with our styles.
This is also a convenience component to stop slots context propagation

Reply:
for an example, please see the new story i added in the PR